### PR TITLE
Optimum-Nvidia launch

### PR DESCRIPTION
**Pull request by mfuntowicz**

### CODE REVIEW

File: optimum-nvidia-launch.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+The FP8 flag (shown above) uses a predefined calibration strategy by default, though you can provide your own calibration dataset and customized tokenization to tailor the quantization to your use case.
+For more details, check out our [documentation](https://github.com/huggingface/optimum-nvidia).
+
+
+### Performance Evaluation
+
+When evaluating the performance of an LLM, we consider 2 metrics: First Token Latency and Throughput. 
+First Token Latency (also known as Time to First Token or prefill latency) measures how long you wait from the time you enter your prompt to begin receiving your output, so this metric can tell you how responsive the model will feel. 
+Optimum-NVIDIA delivers more than 3x faster First Token Latency compared to stock transformers:
```
Comment (laikhtewari) [inline optimum-nvidia-launch.md:70]: up to 3.3x faster

File: optimum-nvidia-launch.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+<br>
+<figure class="image">
+  <img alt="" src="assets/optimum_nvidia/first_token_latency.svg" />
+  <figcaption>Figure 1. Time it takes to generate the first token (ms)</figcaption>
+</figure>
+<br>
+
```
Comment (laikhtewari) [inline optimum-nvidia-launch.md:78]: remove comma after tokens

File: optimum-nvidia-launch.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+Optimum-NVIDIA delivers up to 28x better throughput compared to stock transformers:
+
+<br>
+<figure class="image">
+  <img alt="" src="assets/optimum_nvidia/throughput.svg" />
+  <figcaption>Figure 2. Throughput (token / second)</figcaption>
+</figure>
+<br>
+
+Initial evaluations of the recently announced NVIDIA H200 Tensor Core GPU show up to 2x faster inference for LLaMA models compared to an H100.
```
Comment (laikhtewari) [inline optimum-nvidia-launch.md:90]: should be [recently-announced NVIDIA H200 Tensor Core GPU](https://www.nvidia.com/en-us/data-center/h200/)

Comment (laikhtewari) [reply optimum-nvidia-launch.md:90]: with link

Comment (laikhtewari) [reply optimum-nvidia-launch.md:90]: Also needs to be NVIDIA H100 Tensor Core GPU

File: optimum-nvidia-launch.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+- user: mfuntowicz
+---
+
+
+# Optimum-NVIDIA
+## Unlock blazingly fast LLM inference in just 1 line of code
+
+Large Language Models (LLMs) have revolutionized natural language processing and are increasingly deployed to solve complex problems at scale. 
+Achieving optimal performance with these models is notoriously challenging due to their unique and intense computational demands. 
+Optimized performance of LLMs is incredibly valuable for end users looking for a snappy and responsive experience as well as for scaled deployments where improved throughput translates to dollars saved.
```
Comment (jeffboudier) [inline optimum-nvidia-launch.md:16]:
```suggestion
Large Language Models (LLMs) have revolutionized natural language processing and are increasingly deployed to solve complex problems at scale. Achieving optimal performance with these models is notoriously challenging due to their unique and intense computational demands. Optimized performance of LLMs is incredibly valuable for end users looking for a snappy and responsive experience as well as for scaled deployments where improved throughput translates to dollars saved.
```

File: optimum-nvidia-launch.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+That's where Optimum-NVIDIA  comes in. 
+Optimum-NVIDIA is one of the first to allow the user to benefits from the new capabilities added on Ada Lovelace (4090, L40) and Hopper (H100) GPUs, 
+more precisely `float8` accelerated matrix-multiplications, to dramatically accelerates LLM inference.
+
+By changing just a single line of code, you can unlock up to **28x faster** inference on the NVIDIA platform
```
Comment (jeffboudier) [inline optimum-nvidia-launch.md:22]:
```suggestion
That's where Optimum-NVIDIA comes in. Available on Hugging Face, Optimum-NVIDIA dramatically accelerates LLM inference on the NVIDIA platform through an extremely simple API. By changing **just a single line of code**, you can unlock up to **28x faster inference and 1,200 tokens/second** on the NVIDIA platform.

Optimum-NVIDIA is the first Hugging Face inference library to benefit from the new `float8` format supported on NVIDIA Ada Lovelace and Hopper architectures. FP8, in addition to the advanced compilation capabilities of [NVIDIA TensorRT-LLM](https://developer.nvidia.com/blog/nvidia-tensorrt-llm-supercharges-large-language-model-inference-on-nvidia-h100-gpus/) software, dramatically accelerate LLM inference.
```

File: optimum-nvidia-launch.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
++ from optimum.nvidia.pipelines import pipeline
+```
+
+```python
+# everything else is the same as in transformers!
+pipe = pipeline('text-generation', 'meta-llama/Llama-2-7b-chat-hf', use_fp8=True)
+pipe("Describe a real-world application of AI in sustainable energy.")
+```
+The pipeline interface is great to get up and running quickly, but power users who want fine-grained control (e.g. setting sampling parameters) can use the Model API. 
+You can also enable FP8 with a single flag, which allows you to run a bigger model on a single GPU, at faster speeds, and without sacrificing accuracy.
```
Comment (jeffboudier) [inline optimum-nvidia-launch.md:38]:
```suggestion
You can also enable FP8 quantization with a single flag, which allows you to run a bigger model on a single GPU, at faster speeds, and without sacrificing accuracy. The flag shown in this example uses a predefined calibration strategy by default, though you can provide your own calibration dataset and customized tokenization to tailor the quantization to your use case.

The pipeline interface is great to get up and running quickly, but power users who want fine-grained control over setting sampling parameters can use the Model API.
```

File: optimum-nvidia-launch.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+
+
+```diff
+- from transformers import AutoTokenizer
++ from optimum.nvidia import AutoModelForCausalLM, QuantConfig
```
Comment (jeffboudier) [inline optimum-nvidia-launch.md:43]:
```suggestion
- from transformers import LlamaForCausalLM
+ from optimum.nvidia import LlamaForCausalLM
from transformers import AutoTokenizer
```

Comment (jeffboudier) [reply optimum-nvidia-launch.md:43]: @mfuntowicz please check

File: optimum-nvidia-launch.md [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+
+That's where Optimum-NVIDIA comes in. Available on Hugging Face, Optimum-NVIDIA dramatically accelerates LLM inference on the NVIDIA platform through an extremely simple API. 
+By changing just a single line of code, you can unlock up to **28x faster inference and 1,200 tokens/second** on the NVIDIA platform.
+
+Optimum-NVIDIA is the first Hugging Face inference library to benefit from the new `float8` format supported on NVIDIA Ada Lovelace and Hopper architectures.
+FP8, in addition to the advanced compilation capabilities of [NVIDIA TensorRT-LLM software](https://developer.nvidia.com/blog/nvidia-tensorrt-llm-supercharges-large-language-model-inference-on-nvidia-h100-gpus/) software, dramatically accelerate LLM inference.
```
Comment (jeffboudier) [inline optimum-nvidia-launch.md:20]:
```suggestion
Optimum-NVIDIA is the first Hugging Face inference library to benefit from the new `float8` format supported on NVIDIA Ada Lovelace and Hopper architectures. FP8, in addition to the advanced compilation capabilities of [NVIDIA TensorRT-LLM software](https://developer.nvidia.com/blog/nvidia-tensorrt-llm-supercharges-large-language-model-inference-on-nvidia-h100-gpus/) software, dramatically accelerate LLM inference.
```

File: optimum-nvidia-launch.md [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+```diff
+- from transformers.pipelines import pipeline
++ from optimum.nvidia.pipelines import pipeline
+
+# everything else is the same as in transformers!
+pipe = pipeline('text-generation', 'meta-llama/Llama-2-7b-chat-hf', use_fp8=True)
+pipe("Describe a real-world application of AI in sustainable energy.")
+```
+You can also enable FP8 quantization with a single flag, which allows you to run a bigger model on a single GPU, at faster speeds, and without sacrificing accuracy. 
+The flag shown in this example uses a predefined calibration strategy by default, though you can provide your own calibration dataset and customized tokenization to tailor the quantization to your use case.
```
Comment (jeffboudier) [inline optimum-nvidia-launch.md:35]:
```suggestion
You can also enable FP8 quantization with a single flag, which allows you to run a bigger model on a single GPU, at faster speeds, and without sacrificing accuracy. The flag shown in this example uses a predefined calibration strategy by default, though you can provide your own calibration dataset and customized tokenization to tailor the quantization to your use case.

The pipeline interface is great to get up and running quickly, but power users who want fine-grained control over setting sampling parameters can use the Model API.
```

File: optimum-nvidia-launch.md [hunk 10]
```diff
@@ -0,0 +1,102 @@
...
+```
+
+For more details, check out our [documentation](https://github.com/huggingface/optimum-nvidia)
+
+
+### Performance Evaluation
+
+When evaluating the performance of an LLM, we consider two metrics: First Token Latency and Throughput. 
+First Token Latency (also known as Time to First Token or prefill latency) measures how long you wait from the time you enter your prompt to the time you begin receiving your output, so this metric can tell you how responsive the model will feel. 
+Optimum-NVIDIA delivers up to 3.3x faster First Token Latency compared to stock transformers:
```
Comment (jeffboudier) [inline optimum-nvidia-launch.md:71]:
```suggestion
When evaluating the performance of an LLM, we consider two metrics: First Token Latency and Throughput. First Token Latency (also known as Time to First Token or prefill latency) measures how long you wait from the time you enter your prompt to the time you begin receiving your output, so this metric can tell you how responsive the model will feel. Optimum-NVIDIA delivers up to 3.3x faster First Token Latency compared to stock transformers:
```

File: optimum-nvidia-launch.md [hunk 11]
```diff
@@ -0,0 +1,102 @@
...
+
+<br>
+<figure class="image">
+  <img alt="" src="assets/optimum_nvidia/throughput.svg" />
+  <figcaption>Figure 2. Throughput (token / second)</figcaption>
+</figure>
+<br>
+
+Initial evaluations of the [recently-announced NVIDIA H200 Tensor Core GPU](https://www.nvidia.com/en-us/data-center/h200/) show up to an additional 2x boost in throughput for LLaMA models compared to an NVIDIA H100 Tensor Core GPU. 
+As H200s become more readily available, we will share performance data for Optimum-NVIDIA on an H200.
```
Comment (jeffboudier) [inline optimum-nvidia-launch.md:92]:
```suggestion
Initial evaluations of the [recently-announced NVIDIA H200 Tensor Core GPU](https://www.nvidia.com/en-us/data-center/h200/) show up to an additional 2x boost in throughput for LLaMA models compared to an NVIDIA H100 Tensor Core GPU. As H200s become more readily available, we will share performance data for Optimum-NVIDIA on an H200.
```

File: optimum-nvidia-launch.md [hunk 12]
```diff
@@ -0,0 +1,102 @@
...
+
+### Next steps
+
+Optimum-NVIDIA currently provides peak performance for the LLaMAForCausalLM architecture + task, so any [LLaMA-based model](https://huggingface.co/models?other=llama,llama2), including fine-tuned versions, should work with Optimum-NVIDIA out-of-the-box today. 
+We are actively expanding support to include other text generation model architectures and other tasks, all from within Hugging Face.
```
Comment (jeffboudier) [inline optimum-nvidia-launch.md:97]:
```suggestion
Optimum-NVIDIA currently provides peak performance for the LLaMAForCausalLM architecture + task, so any [LLaMA-based model](https://huggingface.co/models?other=llama,llama2), including fine-tuned versions, should work with Optimum-NVIDIA out-of-the-box today. We are actively expanding support to include other text generation model architectures and other tasks, all from within Hugging Face.
```

File: optimum-nvidia-launch.md [hunk 13]
```diff
@@ -0,0 +1,102 @@
...
+
+
+We continue to push the boundaries of performance and plan to incorporate cutting-edge optimization techniques like In-Flight Batching to improve throughput when streaming prompts and INT4 quantization to run even bigger models on a single GPU.
+
+Give it a try! Starting today we are releasing a first beta version of the [Optimum-NVIDIA repository](https://github.com/huggingface/optimum-nvidia) with instructions how to get started and please share your feedback with us 🤗.
```
Comment (jeffboudier) [inline optimum-nvidia-launch.md:102]:
```suggestion
Starting today we are releasing a first beta version of the [Optimum-NVIDIA repository](https://github.com/huggingface/optimum-nvidia) with instructions on how to get started. Please give it a try and share your feedback with us 🤗!
```

**Review by jeffboudier — APPROVED** [review]
LGTM!

**Pull request merged**